entries/p/payroll.payworks.ca.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Payworks": {
+      "domain": "payroll.payworks.ca",
```

Comment:
As the authentication system seems to cover all of payworks.ca the subdomain is redundant.
```suggestion
      "domain": "payworks.ca",
```